.github/workflows/markdown.yml

```diff
@@ -1,11 +1,6 @@
 ---
 name: Markdown Check
 on:
-  push:
-    paths:
-      - ".github/workflows/*"
-      - ".markdownlint.yaml"
-      - "**/*.md"
```

Comment:
A Pull Request is the only way to push to the main branch, so "on pull request" is sufficient.